src/openrct2-ui/input/MouseInput.cpp

```diff
@@ -612,16 +612,17 @@ static void input_scroll_begin(rct_window* w, rct_widgetindex widgetIndex, Scree
     gTooltipCursorX = screenCoords.x;
     gTooltipCursorY = screenCoords.y;
 
-    int32_t eax, ebx, scroll_area, scroll_id;
+    int32_t scroll_area, scroll_id;
+    ScreenCoordsXY newScreenCoords;
```

Comment:
These `newScreenCoords` are the coordinates on the widget. Perhaps `widgetCoords` would be a more descriptive name. I leave it up for debate though as I guess you might think that was the coordinates of the widget and not the coordinates of the mouse on the widget.

Reply:
I like `scrollCoords` and I think it fits better.